### ISSUE TITLE
Slight conflict between bttv and ffz with 7tv addon

### ISSUE THREAD

**Issue by Nerxxy** [night/betterttv#5875]
**Describe the bug**
With bttv and ffz extensions enabled and the 7tv addon enabled in ffz bttv blocks the "[7tv] _emote_ has been added message"

**To Reproduce**

1. Have bttv and ffz extensions enabled on browser
2. Enable 7tv emotes addon in ffz
3. Add 7tv emote to channel
4. No message appears

**Expected behavior**
Expect to see the message show when adding a 7tv emote

**Screenshots**
easier to show with videos.

**With bttv enabled**

https://user-images.githubusercontent.com/37718547/213567904-a19a150c-063b-48bc-847b-d32a4b55f484.mp4

**Without bttv enabled**

https://user-images.githubusercontent.com/37718547/213568014-5773addb-5f8d-428b-8133-f06a0cecfcfc.mp4

**Desktop (please complete the following information):**

- OS: Windows 10 
- Browser: Opera GX
- Version: 22H2

**Smartphone (please complete the following information):**

- Device: N/A
- OS: N/A
- Browser: N/A
- Version: N/A

**Additional context**
N/A

**Comment by night** [comment]
I don't think this is a conflict, but rather the other extension may be disabling functionality in a compatibility mode. So while it seems working as intended, it sounds like there is a real-time event system for 7TV's service too that could be supported on our end.
